pkgs/servers/zigbee2mqtt/update.sh

```diff
@@ -1,30 +1,32 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -i bash -p nodePackages.node2nix nodejs-12_x curl jq nix-update
+#! nix-shell -I nixpkgs=../../.. -i bash -p nodePackages.node2nix curl jq nix-update common-updater-scripts
 
-CURRENT_VERSION=$(nix eval --raw '(with import ../../.. {}; zigbee2mqtt.version)')
-TARGET_VERSION=$(curl https://api.github.com/repos/Koenkk/zigbee2mqtt/releases/latest | jq -r ".tag_name")
-ZIGBEE2MQTT=https://github.com/Koenkk/zigbee2mqtt/raw/$TARGET_VERSION
+set -euo pipefail
+
+CURRENT_VERSION=$(nix eval -f ../../.. --raw zigbee2mqtt.version)
+TARGET_VERSION="$(curl https://api.github.com/repos/Koenkk/zigbee2mqtt/releases/latest | jq -r ".tag_name")"
 
 if [[ "$CURRENT_VERSION" == "$TARGET_VERSION" ]]; then
     echo "zigbee2mqtt is up-to-date: ${CURRENT_VERSION}"
     exit 0
 fi
 
+ZIGBEE2MQTT=https://github.com/Koenkk/zigbee2mqtt/raw/$TARGET_VERSION
 curl -LO $ZIGBEE2MQTT/package.json
 curl -LO $ZIGBEE2MQTT/npm-shrinkwrap.json
 
-node2nix --nodejs-12 \
-  -l npm-shrinkwrap.json \
-  -c node.nix \
-  --bypass-cache \
+node2nix \
+  --composition node.nix \
+  --lock npm-shrinkwrap.json \
+  --development \
   --no-copy-node-env \
-  --node-env ../../development/node-packages/node-env.nix
-rm package.json npm-shrinkwrap.json
+  --node-env ../../development/node-packages/node-env.nix \
+  --nodejs-14 \
```

Comment:
https://nodejs.org/en/about/releases/

Let's go with nodejs-16 if we can. Needs to generate the node-env at least.

Reply:
If I run node2nix with the `--nodejs-16` flag, the node version in `node.nix` stays @ 12.
For node 14 it works as intended.

Reply:
I'll take that.